libs/platform/wizard-generator/components/wizard-body/wizard-body.component.ts

```diff
@@ -177,17 +177,17 @@ export class WizardBodyComponent implements OnInit {
     /**
      * @description Array of visible Wizard Steps.
      */
-    // eslint-disable-next-line @typescript-eslint/member-ordering
+
```

Comment:
empty line problems in this file too

Reply:
this was done by the tooling with lint --fix